src/NuGet.Clients/NuGet.VisualStudio.Common/PathValidator.cs

```diff
@@ -85,12 +85,10 @@ public static bool IsValidUncPath(string path)
         /// <param name="url">The url to validate.</param>
         /// <returns>True if valid, False if invalid.</returns>
         [SuppressMessage("Microsoft.Design", "CA1054:UriParametersShouldNotBeStrings", MessageId = "0#", Justification = "We're trying to validate that a stirng is infct a uri")]
+        [Obsolete("Use NuGet.Common.PathValidator.IsValidUrl(string) instead")]
         public static bool IsValidUrl(string url)
         {
-            Uri result;
-
-            // Make sure url starts with protocol:// because Uri.TryCreate() returns true for local and UNC paths even if badly formed.
-            return Regex.IsMatch(url, @"^\w+://", RegexOptions.IgnoreCase | RegexOptions.CultureInvariant) && Uri.TryCreate(url, UriKind.Absolute, out result);
+            return Common.PathValidator.IsValidUrl(url);
```

Comment:
It's in NuGet.VisualStudio.Common, no one depends on it. You can remove it.

Reply:
It is a public API. Can we get into trouble if someone depends on it ?

Reply:
We technically publish a package, but no one downloads it, see https://www.nuget.org/packages/NuGet.VisualStudio.Common/. 

We need to stop pushing these anyways. https://github.com/NuGet/Home/issues/9076. 

The NuGet.VisualStudio extensibility approach is depend on NuGet.VisualStudio, https://www.nuget.org/packages/NuGet.VisualStudio/5.6.0-preview.1.6483

Reply:
Ok, I will remove it.